pkg/tfshim/sdk-v2/provider_diff.go

```diff
@@ -94,6 +94,7 @@ func (p v2Provider) simpleDiff(
 			}
 			state.RawState = priorStateVal
 		}
+
```

Comment:
Can we remove the spurious diff?

Reply:
What do you mean? Care to elaborate?

Reply:
Oh yes the extra line. I'm confused what Diff means here I thought InstanceDiff. Of course.

Reply:
> Avoid spurious diffs caused by schema.NormalizeObjectFromLegacySDK #1330

Oh, I can see why that would be confusing given the PR context. My bad.